python/requirements-build.txt

```diff
@@ -1,4 +1,4 @@
 cython>=0.29
-numpy>=1.16.6
-setuptools>=38.6.0
+oldest-supported-numpy
```

Comment:
AFAIK this shouldn't be added here, this is purely a dependency for the build environment that pip creates

Reply:
Yes, seems like we use this for development.